testdata/test1.esdl

```diff
@@ -22,11 +22,11 @@
         <geometry xsi:type="esdl:Point" CRS="WGS84" lon="4.63726043701172" lat="52.158769628869045"/>
       </asset>
       <asset xsi:type="esdl:GenericProducer" power="5000000.0" id="cf3d4b5e-437f-4c1b-a7f9-7fd7e8a269b4" name="GenericProducer_cf3d">
-        <port xsi:type="esdl:OutPort" id="2d818e3d-8a39-4cec-afa0-f6dbbfd50696" carrier="0bd9cb08-2f69-4e97-8ac8-bd87b07e466a" name="Out" connectedTo="a9793a5e-df4f-4795-8079-015dfaf57f82"/>
         <port xsi:type="esdl:InPort" connectedTo="935fb733-9f76-4a8d-8899-1ad8689a4b12" id="9c258b9d-3149-4720-8931-f4bef1080ec1" carrier="0bd9cb08-2f69-4e97-8ac8-bd87b07e466a_ret" name="In"/>
+		<port xsi:type="esdl:OutPort" id="2d818e3d-8a39-4cec-afa0-f6dbbfd50696" carrier="0bd9cb08-2f69-4e97-8ac8-bd87b07e466a" name="Out" connectedTo="a9793a5e-df4f-4795-8079-015dfaf57f82"/>
         <geometry xsi:type="esdl:Point" CRS="WGS84" lon="4.558639526367188" lat="52.148869383489114"/>
       </asset>
-      <asset xsi:type="esdl:Pipe" related="Pipe1_ret" id="Pipe1" length="6267.0" name="Pipe1" innerDiameter="0.5">
+      <asset xsi:type="esdl:Pipe" related="Pipe1_ret" id="Pipe1" length="6267.0" name="Pipe1" innerDiameter="0.1">
```

Comment:
Why is the diameter change required?

Reply:
Diameter change was, for other tests I decreased it to have a more relevant use case with higher velocities.